website/static/js/filepage/index.js

```diff
@@ -147,8 +147,8 @@ var FileViewPage = {
         //Ran into problems with mithrils caching messing up with multiple "Panels"
         self.revisions = m.component(FileRevisionsTable, self.file, self.node, self.enableEditing, self.canEdit);
         self.revisions.selected = false;
-        self.revisions.title = 'Revisions';
-
+        self.revisions.title = 'More';
```

Comment:
This can be done in a separate PR when it becomes necessary; for the time being, "Revisions" is more suitable.